base_phone_rate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_phone_rate_user,phone.rate.user,model_phone_rate,base.group_user,1,1,1,1
```

Comment:
Regular users shouldn't have write access on this.
```suggestion
access_phone_rate_user,phone.rate.user,model_phone_rate,base.group_system,1,1,1,1
```

Now, we need a master `vertical_isp` App module creating the basic Groups, "ISP Manager" and "ISP User", and then revisit this.